src/modules/AddOnProduct.rb

```diff
@@ -2235,7 +2235,7 @@ def add_product_from_cd(url, pth, priority, prodname, check_name)
         end
 
         # ask for a different medium
-        current_url = AskForCD(current_url, prodname)
+        current_url = AskForCD(url, prodname)
```

Comment:
how can it be fixed when current_url equal url? Only possibility I see is that `AddRepo` modify its parameter, so maybe this is actually bug? Maybe some refactoring break behavior of method?